javascript/src/stories/components/BasicCourseSearch/CourseFilters.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import CourseFilters from "main/components/BasicCourseSearch/CourseFilters";
+export default {
+  title: 'components/BasicCourseSearch/CourseFilters',
+  component: CourseFilters
+};
+
+
+
+const Template = (args) => <CourseFilters {...args} />;
+
+
+export const AllUnchecked = Template.bind({});
+AllUnchecked.args = {
+    cancelled:false,
+    handleCancelledOnChange: ()=>{},
```

Comment:
Instead of `() => {}` maybe try something like `() => action('cancelled button click!')`. This way you have confirmation in storybook that your component is registering events correctly. Same for `handleClosedOnChange` and `handleFullOnChange`

Reply:
This actions documentation may be of use  https://github.com/storybookjs/storybook/blob/master/addons/actions/ADVANCED.md

Reply:
It did not show the action in storybook. Thus, we kept the origin code. Btw, thanks Steven for offering that suggestion!

Reply:
Are we required to have these fields be populated for Storybook to work? If this is just going to do nothing, can we remove this line entirely?

(Same goes for lines 18 and 20)

Reply:
Maybe we should keep those fields as a place for folks to do more work on this storybook component in a future commit, along the lines that Steven suggests.

Reply:
On further reflection, I agree with removing this.  I've made a separate branch where I've tried some things, but I was unable to get Steven's suggestion of actions to work with this code.